content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -80,91 +80,91 @@ <h2>HTML Sectioning Elements</h2>
           </tbody>
         </table>
       </section>
 
       <section>
-        <h2>General Principles of Landmark Design</h2>
-        <p>Including <strong>all perceivable content</strong> on a page in one of its landmark regions and giving each landmark region a semantically meaningful role is one of the most effective ways of ensuring assistive technology users will not overlook information that is relevant to their needs.</p>
-        <p><strong>Step 1: Identify the logical structure</strong></p>
+        <h2>ランドマークデザインの一般的な原則</h2>
+        <p>ページ上の<strong>すべての知覚可能なコンテンツ</strong>をランドマーク領域のいずれかに含め、各ランドマーク領域に意味的に意味のある役割を与えることは、支援技術の利用者が自分たちのニーズに関連する情報を見逃さないようにする最も効果的な方法の一つです。</p>
+        <p><strong>ステップ1: 論理構造を特定する</strong></p>
 
         <ul>
-          <li>Break the page into perceivable areas of content which designers typically indicate visually using alignment and spacing.</li>
+          <li>ページを、デザイナーが通常、整列と間隔を使用して視覚的に示す知覚可能なコンテンツの領域に分割します。</li>
 
-          <li>Areas can be further defined into logical sub-areas as needed.</li>
+          <li>領域は必要に応じて論理的なサブエリアにさらに定義することができます。</li>
 
-          <li>An example of a sub-area is a portlet in a portal application.</li>
+          <li>サブエリアの例として、ポータルアプリケーションのポートレットがあります。</li>
         </ul>
 
-        <p><strong>Step 2: Assign landmark roles to each area</strong></p>
+        <p><strong>ステップ2: 各エリアにランドマークロールを割り当てる</strong></p>
 
         <ul>
-          <li>Assign landmark roles based on the type of content in the area.</li>
+          <li>エリア内のコンテンツのタイプに基づいてランドマークロールを割り当てます。</li>
 
-          <li><code>banner</code>, <code>main</code>, <code>complementary</code> and <code>contentinfo</code> landmarks should be top level landmarks.</li>
+          <li><code>banner</code>、<code>main</code>、<code>complementary</code>、<code>contentinfo</code>のランドマークは、トップレベルのランドマークであるべきです。</li>
 
-          <li>Landmark roles can be nested to identify parent/child relationships of the information being presented.</li>
+          <li>ランドマークロールは、提示される情報の親子関係を識別するためにネストすることができます。</li>
         </ul>
 
-        <p id="aria_lh_step3"><strong>Step 3: Label areas</strong></p>
+        <p id="aria_lh_step3"><strong>ステップ3: エリアにラベルを付ける</strong></p>
 
         <ul>
           <li>
-            If a specific landmark role is used more than once on a page, provide each instance of that landmark with a unique label.
-            There is one rare circumstance where providing the same label to multiple instances of a landmark can be beneficial: the content and purpose of each instance is identical.
-            For example, a large search results table has two sets of identical pagination controls -- one above and one below the table, so each set is in a navigation region labelled <q>Search Results</q>.
-            In this case, adding extra information to the label that distinguishes the two instances may be more distracting than helpful.
+            特定のランドマークロールがページ上で複数回使用される場合、そのランドマークの各インスタンスに固有のラベルを提供します。
```

Comment:
ようするにIDということですが、これでどうでしょう？

```suggestion
            特定のランドマークロールがページ上で複数回使用される場合、そのランドマークの各インスタンスに一意なラベルを提供します。
```